lucille-core/src/main/java/com/kmwllc/lucille/stage/Print.java

```diff
@@ -48,30 +52,38 @@ public Print(Config config) {
     this.shouldLog = config.hasPath("shouldLog") ? config.getBoolean("shouldLog") : true;
     this.excludeFields = config.hasPath("excludeFields") ? config.getStringList("excludeFields") : null;
     this.overwriteFile = config.hasPath("overwriteFile") ? config.getBoolean("overwriteFile") : true;
-    this.appendThreadName = ConfigUtils.getOrDefault(config, "appendThreadName", false);
+    this.appendThreadName = ConfigUtils.getOrDefault(config, "appendThreadName", true);
 
-    if (appendThreadName && outputFilePath == null) {
-      log.warn("Print Stage configured to have no outputFile but separateThreads = true. separateThreads has no effect.");
+    // if appendThreadName is *explicitly* set to true in the config, but no output file, warn it has no effect.
+    if ((config.hasPath("appendThreadName") && config.getBoolean("appendThreadName")) && outputFilePath == null) {
+      log.warn("appendThreadName was set to true in Print Config, but no outputFile was specified. appendThreadName has no effect.");
     }
   }
 
   public void start() throws StageException {
-    if (outputFilePath != null) {
+    // create the file writer, if needed, if we aren't using appendThreadName
+    if (outputFilePath != null && !appendThreadName) {
       try {
-        if (appendThreadName) {
-          String appendedOutputFilePath = FileUtils.appendStringToFileName(outputFilePath, Thread.currentThread().getName());
-          writer = new BufferedWriter(new FileWriter(appendedOutputFilePath, !overwriteFile));
-        } else {
-          writer = new BufferedWriter(new FileWriter(outputFilePath, !overwriteFile));
-        }
+        this.writer = new BufferedWriter(new FileWriter(outputFilePath, !overwriteFile));
```

Comment:
consider moving this into processDocument so the writer is always created in the same place regardless of whether we're using appendThreadName. Since we're forced to _sometimes_ do it in processDocument it might be cleaner to _always_ do it there